packer/packer.go

```diff
@@ -52,7 +52,7 @@ func New(
 }
 
 // Schedule schedule a packing flow to pack new block upon given parent and clock time.
-func (p *Packer) Schedule(parent *chain.BlockSummary, nowTimestamp uint64) (flow *Flow, err error) {
+func (p *Packer) Schedule(parent *chain.BlockSummary, nowTimestamp uint64, minTxPriorityFee uint64) (flow *Flow, err error) {
```

Comment:
`minTxPriorityFee` to be a property of `Packer` and can be access by flow via `f.packer.minTxPriorityFee`

Reply:
`minTxPriorityFee` is only used by `flow.go` and its constructor `newFlow` is private and called from here. The rest of the `packer` does not use it so I would argue that is rather a `Flow` property.

Reply:
Yes, agree with you that it is Flow only variable. I am thinking the way to ease our life. By adding it to `Flow`'s parent `Packer`, we don't need to pass it every time the `Schedule` was called. Took a look into the code structure, I think it's better to be inside Packer and let Flow inherit that property.

Reply:
Well, if you put it in the `packer` you need to pass it to the constructor. [The original version of this code had it there actually and I changed it to this](https://github.com/vechain/thor/pull/988/commits/9d51934c004c218ef6b493d469cfd2cc4162c6cc). You can see that in terms of changes it is pretty much the same, so I do not think it makes things easier tbh.

Reply:
I am more into your initial version as `Schedule` doesn't need to know about `effective priority fee`. Makes more sense to me when `effective priority fee` becomes `Packer`'s config property.

Reply:
Done